packages/build/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "@automattic/jetpack-build",
+	"version": "1.0.0",
+	"description": "Jetpack build scripts",
+	"main": "index.js",
+	"bin": {
+		"jetpack-build": "build-module-assets.js"
+	},
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1"
+	},
+	"author": "Daniel Walmsley <dan.walmsley@automattic.com> (http://goldsounds.com)",
+	"license": "GPL-2.0-or-later",
+	"devDependencies": {
+		"@automattic/calypso-build": "^3.0.0",
```

Comment:
What do you think if we'd drop Gulp wrappers out from the equation? 🙂  That would be really cool step towards a more simple setup.

I think it would be much cheaper in terms of future code maintenance to just rely directly in Webpack instead of having an additional layer in between. I don't really see anything extra here in the build script that adds extra value unless I'm missing something?

Ok of course if you'd want to have separate `@automattic/jetpack-build` available e.g. for VaultPress, but again I think it would be more simple to consolidate on `@automattic/calypso-build` (which I think we should rename to just `@automattic/build` but that's minor details).

Thoughts? Happy to brainstorm a bit how it would look like practically.